stm32cube/stm32f4xx/CMakeLists.txt

```diff
@@ -24,11 +24,7 @@ zephyr_library_sources_ifdef(CONFIG_USE_STM32_HAL_DMA drivers/src/stm32f4xx_hal_
 zephyr_library_sources_ifdef(CONFIG_USE_STM32_HAL_DMA2D drivers/src/stm32f4xx_hal_dma2d.c)
 zephyr_library_sources_ifdef(CONFIG_USE_STM32_HAL_DMA_EX drivers/src/stm32f4xx_hal_dma_ex.c)
 zephyr_library_sources_ifdef(CONFIG_USE_STM32_HAL_DSI drivers/src/stm32f4xx_hal_dsi.c)
-if(CONFIG_ETH_STM32_HAL_API_V2)
-  zephyr_library_sources(drivers/src/stm32f4xx_hal_eth.c)
-else()
-  zephyr_library_sources_ifdef(CONFIG_USE_STM32_HAL_ETH drivers/src/Legacy/stm32f4xx_hal_eth.c)
-endif()
+zephyr_library_sources_ifdef(CONFIG_USE_STM32_HAL_ETH drivers/src/stm32f4xx_hal_eth.c)
```

Comment:
Existing code should be kept.
We'll remove V1 support in another PR.

Reply:
Done